cves/kernel/CVE-2016-8650.yml

```diff
@@ -305,23 +316,27 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: true
+  any_discussion: true
+  note: |
+    the developers that found the bug actually brought it to redhat in June
+    of that year, and after being dissatisfied with redhat not putting out a fix
+    in time, they released information about the bug
```

Comment:
Probably should add this link where the bug was disclosed https://seclists.org/fulldisclosure/2016/Nov/76